wordpress/README.md

```diff
@@ -17,13 +33,15 @@ This template describes a **fault tolerant** and **scalable** Wordpress environm
 1. Wait until the stack reaches the state **CREATE_COMPLETE**
 1. Grab the `URL` of the Wordpress environment from the **Outputs** tab of your stack.
 
-### Dependencies
-* `vpc/vpc-*azs.yaml` (**required**)
-* `vpc/vpc-ssh-bastion.yaml`
-* `security/auth-proxy-*.yaml`
+## Dependencies
+* `vpc/vpc-2azs.yaml` (**required**)
```

Comment:
In the other templates it picks Subnets A and B. As long as exporting multiple values is not possible that's a workaround? But it still runs in other vpcs as well? It just dosn#t mak use of all Subnets.